kiwixbuild/flatpak_builder.py

```diff
@@ -49,6 +49,8 @@
         "--socket=pulseaudio",
         "--share=network",
         "--share=ipc",
+        "--filesystem=host:ro",
+        ""
```

Comment:
Doesn't this extra empty argument cause any issues?

Reply:
I will remove this line and merge